mandatory/3-playing-computer.js

```diff
@@ -28,7 +28,6 @@ const f2 = function (a, b) {
 
 console.log(x);
 console.log(a);
-console.log(b);
```

Comment:
Well done Soraiya, just looking as the common way to solve the issue. However you could assign B a value of zero and you would have had similar outcome. Truth is, we are not 100% sure of if there was any B or if it has been missed.
Overall great code.